hardly/jobs.py

```diff
@@ -36,11 +36,7 @@ def process_message(
                 logger.debug(f"{topic} not in {topics}")
                 return []
 
-        event_object: Any
-        if source == "centosmsg":
```

Comment:
oh i see now, i was wondering how come we don't need it :D

Reply:
This is a relic (copied from packit-service worker) from handling stream repositories on git.centos.org.
We're not going to handle those in hardly and it'll eventually be removed also from packit-service.